internal/cli/atlas/serverless/watch.go

```diff
@@ -53,22 +53,22 @@ func (opts *WatchOpts) Run() error {
 	return opts.Print(nil)
 }
 
-// mongocli atlas serverlessCluster(s) watch <clusterName> [--projectId projectId].
+// mongocli atlas serverless|sl watch <instanceName> [--projectId projectId].
 func WatchBuilder() *cobra.Command {
 	opts := &WatchOpts{}
 	cmd := &cobra.Command{
-		Use:   "watch <clusterName>",
-		Short: "Watch for a serverless cluster to be available.",
+		Use:   "watch <instanceName>",
+		Short: "Watch for a serverless instance to be available.",
```

Comment:
Suggestion: If possible, its better to use different words to describe the command in case someone still doesn't understand what "watch" means. How do you feel about:
```suggestion
		Short: "Monitor the status of serverless instance.",
```

Question: The other serverless commands have a required role, does this one too? If so, we should add it in a long description after line 61.